README.md

```diff
@@ -2,6 +2,10 @@
 
 A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
+## Link to Project 
+
+[Click Me Please 👋](https://budget01.herokuapp.com/)
+
```

Comment:
As required, The project should be deployed properly, the home and detail page should work as expected.

![Screenshot (539)](https://user-images.githubusercontent.com/66112760/157745183-435e2c2d-3f6a-42c2-97f9-555c201cb187.png)

Currently, I am unable to sign up with the deployed version of the app. Kindly fix this 👍